lisa/sut_orchestrator/azure/tests/azure_locations_westus2.json

```diff
@@ -4,140 +4,117 @@
     "capabilities": [
         {
             "location": "westus2",
-            "vm_size": "Standard_M208ms_v2",
+            "vm_size": "Standard_D8a_v3",
             "capability": {
                 "type": "requirement",
-                "name": "westus2_Standard_M208ms_v2",
-                "isDefault": false,
+                "name": "westus2_Standard_D8a_v3",
+                "is_default": false,
                 "node_count": 1,
-                "core_count": 208,
-                "memory_mb": 5836800,
+                "core_count": 8,
+                "memory_mb": 32768,
                 "disk_count": {
                     "min": 0,
-                    "max": 64,
-                    "max_inclusive": true
-                },
-                "nic_count": {
-                    "min": 0,
-                    "max": 8,
+                    "max": 16,
                     "max_inclusive": true
                 },
+                "nic_count": 1,
                 "gpu_count": 0,
                 "features": {
-                    "is_allow_set": true
```

Comment:
Can we make it "allows_set"? @WenyiQ CC

Reply:
Yeah I also feel like "allow_set" would be better.

Reply:
Yeah, no s, typo on my end :)

Reply:
The `is` make it clear that the field "is an allow set", "is not a denied set". It's a common practice that, if the value is boolean. But the "allow_set" may mean "allow to set something", it's ambiguous.

Reply:
Being boolean is not implicitly telling you the same story? It either is or not that thing you're referring to.